### PR TITLE
Add works and about myself

### PR DESCRIPTION
**Pull request by shasherazi**
Added the following:
- Featured work
- Recent works
- About myself
- Skills

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -76,3 +108,196 @@ h2 {
   text-align: center;
   list-style: none;
 }
+
+.recent-works-section {
+  margin: 120px 24px 0 24px;
+}
```
Comment (Olamarx) [inline src/style.css:114]: - [ ] Kindly style this section using grid as it would would be needed for the desktop version.

File: src/index.html [hunk 2]
```diff
@@ -56,6 +56,186 @@ <h2 class="headline-supporting-text white-text">
           </li>
         </ul>
       </section>
+
+      <section class="recent-works-section">
+        <h2 class="recent-works-heading">My Recent Works</h2>
+        <hr class="recent-works-hr" />
+        <div class="recent-works-grid">
+          <section class="recent-works-featured">
```
Comment (Olamarx) [inline src/index.html:64]: [OPTIONAL] An article should only be used inside a section. I would advise you change all the section inside the whole `"recent-works-section` class to article.

File: src/index.html [hunk 3]
```diff
@@ -56,6 +56,186 @@ <h2 class="headline-supporting-text white-text">
...
+                <li class="recent-works-card-tag">Ruby</li>
+              </ul>
+              <button class="recent-works-card-btn btn">See Project</button>
+            </section>
+          </section>
+        </div>
+      </section>
+
+      <section class="about">
+        <section class="about-about">
```
Comment (Olamarx) [inline src/index.html:187]: [OPTIONAL] Kindly apply the comment about the article tag to this also.

File: src/index.html [hunk 4]
```diff
@@ -56,6 +56,186 @@ <h2 class="headline-supporting-text white-text">
...
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <button class="about-btn btn">Get My Resume</button>
+          <hr class="about-hr" />
+        </section>
+
+        <section class="about-skills white-text">
```
Comment (Olamarx) [inline src/index.html:199]: - [ ] In this section, The technology categories ("Languages", "Frameworks" and "Skills") should be in a ul, each of them in a li tag and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. This is to make the listing orderly.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @shasherazi, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Readme file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (shasherazi) [reply src/style.css:114]: Hello, I actually style a div just two lines below as a grid. I just left the heading and the hr out. Do I need to include these in the grid too? They feel like they should be separated from the rest of the elements. The cards can be in a grid, and the heading can be separate? Please guide me a bit @Olamarx.

File: src/index.html (hunk 4 above)
Comment (shasherazi) [reply src/index.html:199]: I'm a bit confused about this.

This is how my hierarchy looks right now without the CSS.
image

The skills section is in an `article`. The headings "Languages", "Frameworks" and "Skills" are in an `li` with class `.about-skills-list-item` and the further skills like  "JavasScript", "HTML" and "CSS" are in their own `li`s.

I might be failing to understand something. I'd be very grateful if you could please explain once again in a simple way what changes I need to make @Olamarx

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @shasherazi,

I am Favour 👩🏽 , your reviewer for this project milestone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**